go/common/crash/crash.go

```diff
@@ -54,14 +65,18 @@ func newDefaultRandomProvider() RandomProvider {
 }
 
 func defaultCrashMethod() {
-	runtime.Breakpoint()
+	// TODO: is there anything wrong with just using os.Exit here?
```

Comment:
As long as this exits immediately without giving the program a way to cleanup, this is fine.

Reply:
In case of non-zero exit code it just does `syscall.Exit(code)`

Reply:
Just use `os.Exit(code)`, using the syscall package is frowned upon unless absolutely required.

Reply:
Yeah in this case just remove the comment.